File: results/conditionals.java

public class Conditionals {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        // Data initialization
        int NUM1 = 25;
        int NUM2 = 15;
        int NUM3 = 25;
        int NUM4 = 15;
        int NEG_NUM = -1234;
        String CLASS1 = "ABCD";
        int CHECK_VAL = 65;
        String PASS = "PASS";
        String FAIL = "FAIL";

        // Conditional Statements
        if (NUM1 > NUM2) {
            System.out.println("IN LOOP 1 - IF BLOCK");
        } else {
            System.out.println("IN LOOP 1 - ELSE BLOCK");
        }

        if (NUM3 == NUM4) {
            System.out.println("IN LOOP 2 - IF BLOCK");
        } else {
            System.out.println("IN LOOP 2 - ELSE BLOCK");
        }

        // Custom Conditions
        if (CHECK_VAL >= 41 && CHECK_VAL <= 100) {
            System.out.println("PASSED WITH " + CHECK_VAL + " MARKS.");
        } else if (CHECK_VAL >= 0 && CHECK_VAL <= 40) {
            System.out.println("FAILED WITH " + CHECK_VAL + " MARKS.");
        }

        // Switch Statement
        switch (NUM1) {
            case 2:
                System.out.println("NUM1 LESS THAN 2");
                break;
            case 19:
                System.out.println("NUM1 LESS THAN 19");
                break;
            default:
                System.out.println("NUM1 LESS THAN 1000");
                break;
        }
    }

File: results/conditionals.py
# Identification Division
program_name = "CONDITIONALS"

# Data Division
# Working-Storage Section
num1 = 10
num2 = 5
num3 = 15
num4 = 20
check_val = 75

# Procedure Division
# Main Logic
if num1 > num2:
    if num3 > num4:
        print("NUM1 is greater than NUM2 and NUM3 is greater than NUM4.")
    else:
        print("NUM1 is greater than NUM2 but NUM3 is not greater than NUM4.")
else:
    print("NUM1 is not greater than NUM2.")

# Custom Predefined Conditions
if check_val >= 41 and check_val <= 100:
    print("CHECK-VAL passed the test.")
elif check_val >= 0 and check_val <= 40:
    print("CHECK-VAL failed the test.")
else:
    print("CHECK-VAL is out of range.")

# SWITCH Statement
true = True
if num1 < 2:
    print("NUM1 is less than 2.")
elif num1 < 19:
    print("NUM1 is less than 19.")
elif num1 < 1000:
    print("NUM1 is less than 1000.")
else:
    print("NUM1 is greater than or equal to 1000.")

# STOP RUN Statement
print("Program execution ended.")